CHANGELOG.md

```diff
@@ -19,3 +19,5 @@ All notable, unreleased changes to this project will be documented in this file.
 - deps: upgraded urllib3 from v1.x to v2.x
 - Fix PAGE_DELETE webhook to include pageType in payload - #17697 by @Jennyyyy0212 and @CherineCho2016
 - Stripe Plugin has been deprecated. It will be removed in the future. Please use [the Stripe App](https://docs.saleor.io/developer/app-store/apps/stripe/overview) instead
+- App Extensions: Added new allowed extension target: NEW_TAB. Once handled in the Dashboard, an extension will be able to open a link in new tab
+- App Extensions: New mount points for Dashboard categories, collections, gift cards, draft orders, discounts, vouchers, pages, pages types and menus
```

Comment:
let's open a PR to docs as well, and later when we merge it, would be good if we edit & link docs here

Reply:
Done in https://github.com/saleor/saleor-docs/pull/1602